### DISCUSSION TITLE
Azure Timeseries Insights will be deprecated in March 2025

### DISCUSSION THREAD

**Discussion by fgheysels** [azure-deprecation/dashboard#207] [Deprecation]
Azure Timeseries Insights is currently in 'sustained engineering mode'.  This means that other then general maintenance and security fixes, no further development is being done.

In March 2025, Timeseries Insights will be deprecated.    
Users are advised to migrate to Azure Data Explorer.   A migration guide can be found [here](https://docs.microsoft.com/en-us/azure/time-series-insights/migration-to-adx)

![image](https://user-images.githubusercontent.com/3605786/158860390-89ea770d-ffe5-403b-8d0c-882c334db24b.png)

(I was informed by the MSFT product team for TSI)

**Comment by tomkerkhove** [comment]
It was on my backlog indeed, sorry for the late report but https://github.com/azure-deprecation/dashboard/issues/208 is open.

15+ more to follow :)
